src/Compilers/CSharp/Test/Emit2/Emit/NumericIntPtrTests.cs

```diff
@@ -11546,6 +11519,27 @@ .maxstack  4
 ");
         }
 
+        [Fact, WorkItem(63860, "https://github.com/dotnet/roslyn/issues/63860")]
+        public void IntPtrOperatorOnNewPlatform()
+        {
+            var source = """
+using System;
+
+class C
+{
+    public static void M()
+    {
+        IntPtr a = default;
+        bool b = a == IntPtr.Zero;
+        a += 1;
+        _ = a + 1;
+    }
+}
+""";
+            var comp = CreateCompilation(source, runtimeFeature: RuntimeFlag.ByRefFields, parseOptions: TestOptions.Regular7_3);
```

Comment:
>runtimeFeature: RuntimeFlag.ByRefFields [](http://example.com/codeflow?start=49&length=39)

Why is this needed?

Reply:
Good catch. That was the wrong runtime feature flag. We need to set the NumericIntPtr feature flag to simulate .NET 7 TFM, so that the test would fail absent the code change.